internal/core/server/workspace.go

```diff
@@ -692,9 +695,7 @@ func (ws *Workspace) DescribeProcesses(ctx context.Context, input *api.DescribeP
 			}
 			output.Processes = append(output.Processes, process)
 		case "container":
```

Comment:
same general feedback as a previous PR: let's move this logic to the respective provider subpackages, even if we hard code dispatch to that logic from here for now.